src/components/Footer.jsx

```diff
@@ -1,9 +1,8 @@
 import React from "react";
 import LogoFooter from "../assets/img/logo.png";
-import notLoggedButtons from './isNotLogged';
+import notLoggedButtons from "./isNotLogged";
```

Comment:
Can you add a rule to EsLint to change double quotes for simple quotes in JavaScript code?